draft-irtf-t2trg-rest-iot.md

```diff
@@ -780,12 +780,17 @@ These include:
 * CoAP security: Section 11 of {{RFC7252}}
 * URI security: Section 7 of {{RFC3986}}
 
-IoT-specific security is mainly work in progress at the time of writing.
-First specifications include:
+IoT-specific security is active area of standardization at the time of writing.
+First finalized specifications include:
 
 * (D)TLS Profiles for the Internet of Things: {{RFC7925}}
+* CBOR Object Signing and Encryption (COSE) {{?RFC8152}}
+* CBOR Web Token {{?RFC8392}}
```

Comment:
When is a spec "finalized"?
I would include the CWT PoP spec: draft-ietf-ace-cwt-proof-of-possession

Reply:
"Finalized" as in published as RFC or publication requested. Assuming the latter will be RFCs by the time this is published as RFC.

Reply:
Added CWT PoP reference